Added myHub Api tests

Aman-Codes opened the pull request:
<!--  Thanks for sending a pull request!  -->

## Proposed changes

Summarize your changes here to communicate with the reviewers and make sure to put the link of the related issue

## How has this been tested:
- [ ] Test environment
    * [ ] GKE
    * [ ] AWS
    * [ ] OpenShift
    * [x] KinD/k3s/k3d cluster
    * [ ] Others
 - [ ] Tested with production environment ( For ChaosCenter Tests )
 - [ ] Have not tested

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus-e2e/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] I have added necessary documentation (if appropriate)
- [x] I have added Screenshots or Terminal snippets in the PR comment to validate the successful execution of changes

**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] data-cy=* parameters have been added in codebase ( ChaosCenter Tests )
- [ ] data-cy=* parameters need to be added in codebase ( ChaosCenter Tests )
- [x] No updates required.

**Code Review**
- [x] Does the test handle fatal exceptions, ie. uncaught exceptions from 3rd party library (if any)

## Special notes for your reviewer:

Out of 23 tests 17 tests passed and 6 tests failed. The six failing tests are:
1. Fetching the experiment manifest from the hub of a project with no access [ Should not be possible ]
2. Fetching all the pre-defined workflows of a project with no access [ Should not be possible ]
3. Fetching the pre-defined experiment manifest from a hub of a project with no access [ Should not be possible ]
4. Deleting the hub of a project with no access [ Should not be possible ]
5. Deleting the hub of a project with viewer access [ Should not be possible ]
6. Syncing the hub of a project with no access [ Should not be possible ]

These tests will pass after the appropriate fix are merged (Tracked on #362)
cc @Jonsy13 

![image](https://user-images.githubusercontent.com/54680709/152092438-f1a462fd-4052-4472-b583-866322503846.png)

Jonsy13 commented on Cypress/cypress/support/commands.js:
```diff
@@ -135,3 +136,190 @@ Cypress.Commands.add("validateScaffold", () => {
+          "project_id": project2Id,
+          "user_id": user2Id
+        }
+      })
```
We can logout user2 here also.

Jonsy13 reviewed (changes requested):
Looks good!, PTAL on minor comments as discussed.

Aman-Codes commented:
@Jonsy13 Changes added in last commit:
- Logged out the user in `securityCheckSetup` function and logging in again before test run
- Moved myhub tests inside `Api_Tests` folder
- Integrated ESLint and Prettier
- Added scripts for linting checks: `npm run lint`, `npm run checkformat`, `npm run format`
- Fixed the warnings and errors  from ESLint and Prettier

Jonsy13 reviewed (approved):
Thanks!!..LGTM
